yarn.lock

```diff
@@ -2707,9 +2707,9 @@
     moment ">=2.14.0"
 
 "@types/node@*":
-  version "13.5.0"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-13.5.0.tgz#4e498dbf355795a611a87ae5ef811a8660d42662"
-  integrity sha512-Onhn+z72D2O2Pb2ql2xukJ55rglumsVo1H6Fmyi8mlU9SvKdBk/pUSUAiBY/d9bAOF7VVWajX3sths/+g6ZiAQ==
+  version "14.14.7"
```

Comment:
Did this intentionally bump our `@types/node` version?

Reply:
I don't know how this happened when all I did was try to update the platformicons package...

Reply:
This is fine. This is only for TS types; which will not end up in the final bundle.

Reply:
We also don't directly depend on the package.